tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1532,4 +1532,88 @@ public function theResponseShouldContainTheFollowingDriveInformation(TableNode $
 			throw new Error('Response is not an array or the array does not consist key "drive"');
 		}
 	}
+
+	/**
+	 * @When user :user gets details of a group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfAGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		try {
+			$groupId = $this->getGroupIdByName($groupName);
+		} catch (Exception $e) {
+			$groupId = WebDavHelper::generateUUIDv4();
+		}
```

Comment:
maybe we should add a comment here to clarify that this is for getting non-existing groups